packages/cashier/src/pages/account-transfer/account-transfer-form/__tests__/account-transfer-form.spec.tsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable testing-library/no-container,testing-library/no-node-access */
 import React from 'react';
-import { fireEvent, render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
why is the order of import changed ?

Reply:
Well, I didn't change the order, maybe eslint did it. I will revert it inline.